src/shared/plan/ScheduledPlanDetails/ScheduledPlanDetails.test.tsx

```diff
@@ -8,7 +8,7 @@ const queryClient = new QueryClient({
   defaultOptions: { queries: { retry: false } },
 })
 
-const wrapper = ({ children }) => (
+const wrapper = ({ children }: { children: React.ReactNode }) => (
```

Comment:
nit: We usually type this guy out as wrapper: React.FC<React.PropsWithChildren>

Reply:
Ah thanks! I followed the wrong pattern - fixed it in other files I saw using the wrong way, too, so others don't mistake that too!